### PR TITLE
New feature tests for new IRC Derwentside

### PR DESCRIPTION
**Pull request by solomonahmed**
## What does this pull request do?

Created a simple feature test for Immigration and Asylum cats of law to test the new PA/AP combination for IRC Derwentside has been correctly added.

## Why make these changes?

We need to add feature tests for new codes to aid testing.

## Checklist

- [x] Provided link to a JIRA ticket: https://dsdmoj.atlassian.net/browse/TA-1264

### CODE REVIEW

File: features/validations/legal_help/immigration_and_asylum/asylum_validation_manual.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Feature: Validation for Immigration and Asylum claims
+
+@delete_outcome_after @manual_submission @valid @derwentside_irc
```
Comment (your) [inline features/validations/legal_help/immigration_and_asylum/asylum_validation_manual.feature:3]: Any reason why you are using `derwentside_irc` and `irc` only in the two other scenarios below?

File: features/validations/legal_help/immigration_and_asylum/immigration_validation_manual.feature [hunk 2]
```diff
@@ -20,6 +20,15 @@ Feature: Validation for Immigration claims
...
             | 712     | IMLB:IOUT | TR001    |           | 01/11/19        | PA20000 | AP20000 | The PA (PA20000) and/or AP (AP20000) fields have been populated with ECF Matter. This value can only be used if the ECF Reference field is also populated.                                                                                             |
 
+    @delete_outcome_after @manual_submission @invalid @irc
+    Scenario Outline: Add invalid Immigration claims using IRC
+        Given user is on their submission details page
+        When user adds an outcome for Immigration with "<case id>", "<mt>", "<ecs code>", "<ecf ref>", "<case start date>", "<pa>" and "<ap>"
+        Then the outcome does not save and the error message "<error message>" appears
+        Examples:
+            | case id | mt        | ecs code | ecf ref  | case start date | pa      | ap      | error message                                                                                       |
+            | 718     | IMXL:IIRC |          |          | 01/11/19        | PA00188 | AP00186 | Access Point - Value AP00186 for the flexfield segment Access Point does not exist in the value set |
```
Comment (your) [inline features/validations/legal_help/immigration_and_asylum/immigration_validation_manual.feature:30]: That's a fancy error message but fair enough 👍

**Review by your — COMMENTED** [review]
Just a question around one of the tags. LGTM otherwise

File: features/validations/legal_help/immigration_and_asylum/asylum_validation_manual.feature (hunk 1 above)
Comment (solomonahmed) [reply features/validations/legal_help/immigration_and_asylum/asylum_validation_manual.feature:3]: Ah.  Good point.  Derwentside is too specific and was what I used first.  I then changed it to irc but obviously forgot to change it in this file.  Thanks!  I'll amend.

File: features/validations/legal_help/immigration_and_asylum/immigration_validation_manual.feature (hunk 2 above)
Comment (solomonahmed) [reply features/validations/legal_help/immigration_and_asylum/immigration_validation_manual.feature:30]: I know.  Tbh I generated the error and then copied the error message into the feature test.  Could probably abbreviate it?  Any thoughts?

File: features/validations/legal_help/immigration_and_asylum/asylum_validation_manual.feature (hunk 1 above)
Comment (solomonahmed) [reply features/validations/legal_help/immigration_and_asylum/asylum_validation_manual.feature]: Now changed. @your

**Review by DeepakTanna — COMMENTED** [review]
Looks good to me.

File: features/validations/legal_help/immigration_and_asylum/immigration_validation_manual.feature (hunk 2 above)
Comment (your) [reply features/validations/legal_help/immigration_and_asylum/immigration_validation_manual.feature:30]: @solomonahmed We should probably review most of the error messages then. Something for another time.

**Review by your — APPROVED** [review]

**Pull request merged**